examples/permissioned-voting/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "local-project-test",
+  "version": "0.0.1",
+  "main": "index.js",
+  "license": "apache-2.0",
+  "devDependencies": {
+    "chai": "^4.2.0"
+  },
+  "dependencies": {
+    "algob": "^0.0.5",
```

Comment:
We can remove `algob`. We are using `yarn link algob` (refer to readme), so `yarn add algob` is not needed.

Reply:
ok